### PR TITLE
readme typos and update

### PR DESCRIPTION
**Pull request by silabs-JulienT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,18 +110,13 @@ bytestream: 25 03 00 00 00
 
 ```
 
-## HOW TO BUILD Z/IP/GATEWAY
```
Comment (rzr) [inline README.md:131]: shouldn´t we keep this chapter title

Comment (silabs-JulienT) [reply README.md:131]: It is the same title as the previous one. Did you have another idea?

Comment (silabs-JulienT) [reply README.md:131]: good point, I thought it was exactly the same title as the previous one.

**Comment by rzr** [comment]
Some remarks to take into account at:

https://github.com/SiliconLabs/zipgateway/pull/25#discussion_r1715426481

File: README.md [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 
-## LICENSING
-
-Z/IP Gateway is covered by one of several different licenses.
-The default license is the [Master Software License Agreement (MSLA)](https://www.silabs.com/about-us/legal/master-software-license-agreement), which applies unless otherwise noted.
-Refer to [LICENSE](./LICENSE) for more details.
+The Z/IP gateway is a free reference design intended for development and demonstration purposes only. It is provided as is, with no warranties. Customers are advised to conduct security and compliance testing of all gateways during the product design process.
 
+> [!WARNING] 
+> The Z/IP Gatreway solution is in maintenance mode. To access a maintained Z-Wave gateway solution supporting the latest protocol features, refer to the [Unify SDK](https://github.com/SiliconLabs/UnifySDK) project.
```
Comment (rzr) [inline README.md]:
```suggestion
> The Z/IP Gateway solution is in maintenance mode. To access a maintained Z-Wave gateway solution supporting the latest protocol features, refer to the [Unify SDK](https://github.com/SiliconLabs/UnifySDK) project.
```

File: README.md [hunk 3]
```diff
@@ -55,8 +45,7 @@ sudo dpkg -L zipgateway # List server and client and other utilities
 cat /usr/local/etc/zipgateway.cfg # To check configuration file
 ```
 
-Use zipgateway using example libzwaveip's reference_client
-(now part of the package previously installed)
+- Use Z/IP Gateway with libzwaveip's reference client.
```
Comment (rzr) [inline README.md]:
```suggestion
- Use Z/IP Gateway with libzwaveip's reference client (part of the package).
```

File: README.md [hunk 4]
```diff
@@ -157,20 +141,17 @@ cmake ..
...
+Feel free to tweak env, debugging using gdb can be helpful too.
 
 ### BUILD ON HOST
 
-To speed up the build, native build can be deported to host using
-different containerization techniques (docker, systemd. chroot, qemu, binfmt),
-check [DevTools](./DevTools/) directory for more.
-
+To speed up the build process, native build can be deported to the host using
+different containerization techniques (docker, systemd. chroot, qemu, binfmt).
```
Comment (rzr) [inline README.md]:
```suggestion
different containerization techniques (docker, systemd, chroot, qemu, binfmt).
```

**Review by rzr — APPROVED** [review]

**Review by rzr — APPROVED** [review]

**Pull request merged**